scripts/missions/rov/1_05_The_Beginning.lua

```diff
@@ -74,7 +74,7 @@ mission.sections =
         [xi.zone.NORG] =
         {
             ['_700']       = mission:event(276):setPriority(1005), -- RoV objectives are highest priority, set higher than progressEvent (1000)
-            ['Comitiolus'] = mission:event(6),
+            ['Comitiolus'] = mission:event(6), -- NPC Chat 12321, not event 6
```

Comment:
What does this mean?

Reply:
Had the wrong item there and made a note to change it, but totally missed.  Replaced with the message received from caps.